pages/ui-components/lifecycle-status.md

```diff
@@ -15,6 +15,7 @@ changelog:
 {% assign component_status_items = site.data.lifecycle-status.components %}
 
 {% assign check_icon_classes = 'text-gray-60' %}
+{% assign converting_pattern_classes = proposal_classes %}
```

Comment:
This should follow the same convention as the other classnames. Below is a suggestion for what to change this line to read as:

```suggestion
{% assign converting_pattern_classes = "lifecycle-bg--convert" %}
```